tests/platform_tests/conftest.py

```diff
@@ -89,6 +89,12 @@ def xcvr_skip_list(duthosts, dpu_npu_port_list):
             logging.debug('Skipping sfp interfaces: {}'.format(sfp_list))
             intf_skip_list[dut.hostname].extend(sfp_list)
 
+        # For Mx topo, skip the SFP interfaces because they are admin down
+        if tbinfo['topo']['name'] == "mx" and hwsku in ["Arista-720DT-G48S4", "Nokia-7215"]:
```

Comment:
hi @lizhijianrd  should we skip the admin down ports by check the admin status other than hard code the port number?

Reply:
Hi @StormLiangMS, I was thinking about that. But I'm not sure if other platform/topo will accept skipping admin down SFP interfaces. Actually SFP tests should work well as long as the SFP module exists. No matter the interface is up or down.
@prgeor What's your suggestion on this? Can you please help review? thanks.